reference/api-reference/anbox-https-api.md

```diff
@@ -742,3 +743,23 @@ Example return value for `curl -s -X GET --unix-socket /run/user/1000/anbox/sock
 See the [VHAL property configuration](https://source.android.com/docs/automotive/vhal/property-configuration) for more information on these fields.
 
 `value_type` is added as a convenience in the Anbox API and maps to the [VHAL property types](https://source.android.com/docs/automotive/vhal/property-configuration#property-types).
+
+(sec-anbox-https-api-metrics)=
+### `/1.0/metrics`
+#### GET
+
+ * Description: Get metrics in Prometheus output data format
+ * Operation: sync
+ * Return: Current metrics in Prometheus output data format
+
+ Return value for `curl -s -X GET --unix-socket /run/user/1000/anbox/sockets/api.unix s/1.0/metrics`:
+
+```bash
```

Comment:
This is not really bash, though. Is that intended?

Reply:
No, it's not. Let me drop that.